asset/css/markbind.css

```diff
@@ -33,6 +33,32 @@ code {
 
 /* Bootstrap small(sm) responsive breakpoint */
 @media (max-width: 767.98px) {
+
+    /* 70% of Bootstrap 4 default font-size for headings */
+    h1 {
```

Comment:
Bootstrap also overrides classes of `h1` (i.e. it does `h1, .h1 {`). Should we consider doing the same?